Documentation/reportgenerationqueries.md

```diff
@@ -164,6 +165,18 @@ spec:
     ORDER BY pod_request_memory_byte_seconds DESC
 ```
 
+## Modifying Columns For Report Display
+
+You can modify the custom resource to display all columns or to hide or show columns as needed. The full endpoint displays all columns.
+An example report for `namespace-cpu-request` in JSON for the full endpoint would be:
+`http://127.0.0.1:8001/api/v1/namespaces/metering/services/http:reporting-operator:http/proxy/api/v2/reports/namespace-cpu-request/full?format=json`
+To use the TableHidden display feature:
+- enter `TableHidden` field in query as true or false
+- run report
+An example report for `namespace-cpu-request` in JSON for the tableHidden endpoint would be:
```

Comment:
Like the earlier comment, this is an example of querying a report, so it would help this was updated to be more accurate.